app/services/obs-importer.ts

```diff
@@ -419,7 +419,12 @@ export class ObsImporterService extends StatefulService<{ progress: number; tota
   importTransitions(configJSON: IOBSConfigJSON) {
     // Only import a single transition from OBS for now.
     // Eventually we should import all transitions
-    if (configJSON.transitions && configJSON.transitions.length > 0) {
+    if (
+      configJSON.transitions &&
+      configJSON.transitions.length > 0 &&
+      // only import if it's a supported transition type
+      Object.values<string>(ETransitionType).includes(configJSON.transitions[0].id)
```

Comment:
The better way to do this would be to use `this.transitionsService.views.getTypes().map(t => t.value)`.  The reason is that available transitions varies by OS, so this will only return the transitions available on the currently running OS.

Reply:
wouldn't we want to avoid importing transitions that aren't supported on the current OS? afaik the only example at the moment is the "Motion" transition, which isn't supported on Mac, but ideally we avoid importing any transition that isn't valid on the currently running OS right?

Reply:
Correct - sorry if it wasn't clear.  Your code isn't restricted to currently available transitions.  The code I provided is restricted to what's available.

Reply:
ahhh, yep, I misunderstood. will update, ty!